solc/CommandLineParser.cpp

```diff
@@ -501,6 +502,7 @@ void CommandLineParser::parseOutputSelection()
 		m_options.compiler.outputs.irOptimized = true;
 		m_options.compiler.outputs.ewasm = true;
 		m_options.compiler.outputs.ewasmIR = true;
+		m_options.compiler.outputs.astCompactJson = true;
```

Comment:
All outputs should be disabled by default. The only reason that some are enabled in assembly mode is that it used to have no output selection at all - everything was always printed to the output. For backwards-compatibility this is still the default behavior in assembly mode but it will change in 0.9.0 (#12370 was already merged into `breaking`).

Reply:
So should I change it back to be on by default?

Reply:
No, it's fine disabled. I was mainly commenting for @r0qs since he seemed unsure if his suggestion was in line with how the CLI is supposed to work.